tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -391,15 +508,15 @@ function doSomething(\Foo\Bar\SomeClass $foo, \Foo\Bar\Buz $buz, \Foo\Bar\Zoof\B
 class SomeClass
 {
     public function doSomething(
-        Ping\Something $something,
-        Pung\Pang $pungpang,
-        Pung $pongpung,
-        Pang\Pung $pangpung,
-        Pyng\Pung\Pong $pongpyngpangpang,
-        Bar\Baz\Buz $bazbuz
+        \Ping\Something $something,
```

Comment:
I think the current fixer is wrong here, removing the leader `\` within the current namespace changes the type that is supposedly used here